README.md

```diff
@@ -79,6 +100,56 @@ Arguments:
 
 <truncated>
 ```
+    
+# Debugging
+Debugging code requires your current `dotnet new3` session to have its active build session configured to DEBUG, and a debug process from Visual Studio to be attached to the current running `dotnet new3` session. The steps required to accomplish this are outlined below.
+
+## Setup
+
+Start by opening up the **Microsoft.Templating.sln** in Visual Studio. This solution contains the projects needed to run, modify & debug the application. Once your solution is loaded, open up a new command prompt and navigate to the root of your repository. At this point, you can take one of two paths.
```

Comment:
Rather than VS specifically, the application that will be attaching the debugger